### ISSUE TITLE
Cannot reproduce the pretrained model accuracy for NTU120 无法复现提供的pretrained模型的accuracy

### ISSUE THREAD

**Issue by kfzyqin** [kenziyuliu/MS-G3D#25]
Hi, 

Thanks for the great work. I follow the given config files to run both the joint and the bone streams for the NTU120 dataset. However, I could not obtain the accuracy as high as the provided pretrained models. 

非常感谢您的好工作. 我用您给的config文件来跑joint和bone stream的NTU120数据集. 但是, 我跑不到像pretrained model那么高的结果. 

For XSub Joint, I got top-1 82.19 and top-5 95.77. However, the results for the pretrained model are: top-1 83.32 and top-5 96.06.
For XSub Bone, I got top-1 84.69 and top-5 96.76. However, the results for the pretrained model are: top-1 85.88 and top-5 97.04. 
For XSet Joint, I got top-1 84.05 and top-5 95.97. However, the results for the pretrained model are: top-1 84.44 and top-5 95.89. 
For XSet Bone, I got top-1 86.47 and top-5 96.89. However, the results for the pretrained model are: top-1 87.39 and top-5 97.36. 

I received consistent worse performance. Can you please enlighten me on how to get the accuracy that is the same as the provided pretrained models? Thank you very much!

**Comment by kenziyuliu** [comment]
Hi Zhenyue,

Thanks for your interest.

1. I would suggest trying the joint-bone ensemble results first, and see if they deviate a lot from those produced by the pre-trained models, as the single-stream performance does have more fluctuations than I expect.  Note also that worse single-stream models sometimes actually give better ensemble results. 
2.  Do pay attention to sensitive hyperparams like LR, full/forward batch sizes, and the number of devices. Have you changed any of these? e.g. a smaller `forward_batch_size` will harm batch norm. 
3. Without further context, I can't really help debugging the performance, and it'd be helpful if you could share the config for those runs (by default they are at the top of `log.txt`; an example [here](https://github.com/kenziyuliu/MS-G3D/issues/14#issuecomment-661611084)).
4. Note also that the pretrained models are the best models during training rather than the last saved models (at the end of training). 
5. The pretrained models are trained using the procedure in README, and, if I remember correctly, mostly with full batch size 32, 1/2 1080ti/2080ti GPUs, and `--amp-opt-level 1`. Apart from those specified in the README, there shouldn't be any additional training configs/procedures/tricks.

Please also have a look at the previous related issues:
  - https://github.com/kenziyuliu/MS-G3D/issues/12
  - https://github.com/kenziyuliu/MS-G3D/issues/14  (also has my config log for NTU120 XSub joint)

Hope this helps! Please feel free to comment below for further questions.
